docs/source/user/fast.farm/InputFiles.rst

```diff
@@ -624,11 +625,58 @@ If DEFAULT is used, then **Mod_Projection=2** when **Mod_Wake=2**,
 and **Mod_Projection=1** otherwise.
 
 
-
-**OutAllPlanes** [-] Output all wake planes in VTK at all time steps. 
-Note: this option requires intensive writing to disk and will drastically slow down the simulation.
+Wake-Added Turbulence (WAT)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The wake-added turbulence model is described in :numref:`FF:WAT`.
+
+**WAT** [switch] Select whether the wake-added turbulence is included.
+There are three options:
+0) no wake-added turbulence,
+1) use predefined turbulence box for the background turbulence,
+2) use a user defined turbulence box.
+When **WAT=1**, the number of points of the box are inferred from the filename, 
+and the dimensions in each directions are taken as :math:`dx=dy=dz=0.03*\text{RotorDiamRef}`.
+
+**WAT_BoxFile**: [quoted string]
+Filepath to the file containing the u-component of the turbulence box 
+(either predefined or user-defined). This turbulence box is expected to be 
+in the Mann box format. 
+The filename should be of the form "Label_1024x32x16.u" where the extension ".u" 
+is required. FAST.Farm will assume that files with extensions ".v" and ".w" are present,
+and these files will also be read. 
+When **WAT=1**, the three digits separated by "x" in the filename are assumed to be 
+the number of points in the x y and z dimension of the box.
```

Comment:
Currently we don't use the ".bin" extension, the extensions are ".u", ".v", ".w", so I think the documentation should be good as it is. 

I've added the precision regarding the quotation and relative/absolute paths. 

There is no predefined dimensions, these can be any dimensions chosen by the user when generating the Mann box. The idea is to push people to use this convention in the filename because Mann Boxes do not store this information anywhere, and it is a critical information to be able to read the box correctly. Most likely, we will provide the user with a bsaeline box with dimensions 512x512x64, so I've changed the description to accomodate for that.

Reply:
I personally think we should get rid of `WAT_NxNyNz` and force the user to use this filename convention.